app/models/user.rb

```diff
@@ -5,5 +5,4 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
-
```

Comment:
@jjwahl how come there is no `validates :user_name, presence: true` here?

Reply:
there is already on the migration file, should be redundant, but nice catch, will add it and commit again!